core/payment/examples/event_api.rs

```diff
@@ -0,0 +1,23 @@
+use bigdecimal::BigDecimal;
```

Comment:
Not really needed... remove or rename to `serialize_event.rs` ?

Reply:
Huh? Rename the import?

Reply:
Ah, I see. You mean the whole file. I'd delete it.

Reply:
oops, haha, was about the whole file